CoverMyDotNet/Models/Common/ActionAttributes.cs

```diff
@@ -0,0 +1,14 @@
+using System;
+using RestSharp.Serializers;
+
+namespace CoverMyDotNet
+{
+	public class ActionAttributes
+	{
+		public string Href { get; set; }
+	    public string Ref { get; set; }
```

Comment:
Thanks for the tip, but I don't think that FxCop works on mac, but I found another program that does the same thing and is a part of mono called Gendarme.